Fix issue where sepa mandate text is incorrectly being displayed (#7303)
stripe/stripe-android: CHANGELOG.md

## XX.XX.XX - 2023-XX-XX

### PaymentSheet

> * [FIXED][7303](https://github.com/stripe/stripe-android/pull/7303) Fixed an issue where SEPA mandate texts were being displayed to payment method forms when they shouldn't be.

Reviewer: ```suggestion
* [FIXED][7303](https://github.com/stripe/stripe-android/pull/7303) Fixed an issue where SEPA mandate texts were being displayed for payment methods when they shouldn't be.
```
